test_scripts/API/SetAudioStreamingIndicator/035_ATF_N_TC_SetAudioStreamingIndicator_media_app_HMI_doesnot_reply_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,79 @@
+-- Requirement summary:
+-- [SetAudioStreamingIndicator] SDL must transfer request from mobile app to HMI in case no any failures
+-- [GeneralResultCodes] GENERIC_ERROR watchdog timeout from HMI
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [HMI_API] SetAudioStreamingIndicator
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- In case media app sends the valid SetAudioStreamingIndicator_request to SDL
+-- and this request is allowed by Policies
+-- SDL must respond with "GENERIC_ERROR, success:false" in case HMI does NOT respond during <DefaultTimeout>
+--
+-- 1. Used preconditions
+-- Allow SetAudioStreamingIndicator RPC by policy
+-- Register and activate media application
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+-- HMI does not reply
+--
+-- Expected result:
+-- SDL->HMI: UI.SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+-- SDL->mobile: SetAudioStreamingIndicator_response("GENERIC_ERROR", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ General Precondition before ATF start ]]
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"}, "SetAudioStreamingIndicator")
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_SetAudioStreamingIndicator_GENERIC_ERROR_audioStreamingIndicator_PAUSE_HMI_doesnot_reply()
+  local corr_id = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", { audioStreamingIndicator = "PAUSE" })
+  :Do(function() -- Do not send HMI response
```

Comment:
@istoimenova Is some reason for colling empty function except visual proposes is script?
If no - @ Istoimenova What is the reasons of calling empty function except visual prompting in script?
If no - I think we should avoid calling empty function in future.